windows/internal/smoke_test.bat

```diff
@@ -3,6 +3,7 @@ set SRC_DIR=%~dp0
 pushd %SRC_DIR%\..
 
 if "%CUDA_VERSION%" == "102" call internal\driver_update.bat
+if "%CUDA_VERSION%" == "110" call internal\driver_update.bat
```

Comment:
Have you checked that the driver is up to date?

Reply:
Thanks for reminding me, and you are right that it is not up to date. It requires >=451.22. I will update this.

Reply:
I believe I will need your help to download the latest driver and upload it to S3 so I can use the link :)